CONTRIBUTING.md

```diff
@@ -0,0 +1,55 @@
+# Contribution guide
+
+The "dom-examples" repository accompany various MDN DOM and Web API documentation pages on [MDN Web Docs](https://developer.mozilla.org).
+This document will help you get started with contributions!
+
+## Types of contribution
+
+> [!NOTE]
+> You can include an example directly in MDN pages using [`{{EmbedLiveSample()}}` macros](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Page_structures/Macros/Commonly_used_macros#code_samples) or regular Markdown code blocks.
+> These methods are simpler to maintain as the code lives beside the rest of the content.
+> Only add examples to this repository if your example doesn't easily run on MDN pages for technical or security reasons.
+
+There are many ways you can help improve this repository! For example:
+
+- **Write a new example.** If you would like to add a new example, make sure this feature is supported by a stable version of modern browsers.
+- **Improve an existing example.** You could make improvements or add some other changes which would make example more helpful to the users.
+- **Fix a bug:** we have a list of [issues](https://github.com/mdn/dom-examples/issues), or maybe you found your own.
+
+## Getting started
+
+You will need to have Git and GitHub set up to be able to contribute.
+
+### Set up Git and GitHub
+
+For more information on setting up Git on your machine, [read this article](https://help.github.com/articles/set-up-git/).
+With the above dependencies satisfied, [create your new account on Github](https://github.com/join).
+
+Next up, you need to fork and clone the repo to be able to contribute to it.
+You can [learn about forking on Github](https://help.github.com/articles/fork-a-repo).
```

Comment:
```suggestion
With the above dependencies satisfied, [create your new account on GitHub](https://github.com/join).

Next up, you need to fork and clone the repo to be able to contribute to it.
You can [learn about forking on GitHub](https://help.github.com/articles/fork-a-repo).
```

Just nitpicking, really :)

Reply:
"GitHub" with a lowercase H infraction! 😱 Thank you!